src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -87,6 +89,10 @@ public function __construct(
             throw new InvalidArgumentException('The "mode" parameter value is not valid.');
         }
 
+        if (null !== $options && [] !== $options) {
```

Comment:
```suggestion
        if ($options) {
```

Maybe? Same for other occurrences. If I remember correctly, Nicolas tends to recommend this notation to have better performance and less code noise